README.md

```diff
@@ -1,18 +1,680 @@
+
+```json
+{
+  "file_name": "foundations.md", // name of the file that gets written to the docs directory
+  "file_dir": "dev-guide/golang", // the directory that the file is written to
+  "id": "foundations", // id of the document that is used by Docusaurus
+  "slug": "/dev-guide/go/foundations", // slug of the document that is used by Docusaurus - OVERRIDES id + file_dir slug
+  "title": "Go SDK developer's guide - Foundations", // title of the published document
+  "sidebar_label": "Foundations", // sidebar label of the document
+  "sidebar_position": 1, // position of the label in the sidebar
+  "description": "The Foundations section of the Temporal Go SDK Developer's guide covers the minimum set of concepts and implementation details needed to build and run a Temporal Application in Go – that is, all the relevant steps to start a Workflow Execution that executes an Activity.", // Published description metadata
+  "use_description": false, // uses the description as the first sentence in the generated doc
+  "toc_max_heading_level": 4, // sets the maximum toc level for the generated doc
+  "add_tabs_support": true, // adds mdx imports to generated doc to support tabs
+  "sections": [ // array of sections
+    {
+      "type": "p", // section type - p sections use the previous heading as an anchor
+      "id": "go/foundations" // this is the directory + doc id within docs-src
+    },
+    {
+      "type": "h2", // can go as deep as h4
+      "id": "clusters/how-to-install-temporal-cli"
+    },
+    {
+      "type": "h2",
+      "id": "go/add-sdk"
+    },
+    {
+      "type": "p",
+      "id": "go/how-to-use-the-go-sdk"
+    },
+    {
+      "type": "h3",
+      "id": "go/api-reference-go"
+    },
+    {
+      "type": "h3",
+      "id": "go/code-samples"
+    },
+}
+```
+
+### How to use DACX
+
+**What is DACX?**
+
+DACX is short for “Docs as code”.
+All the logic of the DACX tool is built into the Assembly Workflow.
+
+**Why use DACX?**
+
+1. The tool is similar [Snipsync](https://github.com/temporalio/snipsync), but with DACX you write documentation in the same file as the code. As a code sample developer, you can write the documentation inline with the code and then use the docs Assembly Workflow to generate Markdown guides from those code samples. However, rather than curating the information and code in the Markdown file within the documentation repo like you do with Snipsync, you curate the information within the source code file. As with both tools, you need to work within both repositories to manufacture the end result.
+
+2. DACX provides rich content in multiple places for the next big training event and is suited to meet our long term LLM goals. This is because DACX source files contain parts of the narrative inline with the code and the published guides will put the full narrative together, each complimenting the other. During the next training event, LLMs will consume both DACX source files and the generated guides.
```

Comment:
```suggestion
DACX is similar to [Snipsync](https://github.com/temporalio/snipsync), except that DACX lets you write the documentation within the code. 
Rather than curating the information and code within the documentation repository, you curate the information within the source code's repository. 
The Assembly Workflow then generates Markdown guides from the narrative inlined with the code.
As a result, DACX provides rich content in multiple places. 
```

Reply:
Nice!